cirq/contrib/quimb/mps_simulator.py

```diff
@@ -231,92 +235,243 @@ def sample(
         return np.array(measurements, dtype=int)
 
 
+def _sum_reduce(M, ind):
+    # TODO(tonybruguier): Use M.sum_reduce(ind, inplace=True) once
```

Comment:
interesting that this doesn't exist (yet, apparently). How is this different than the partial trace? Can you just manipulate the indices' names so they will be contracted away and do the sum you want? Or insert an identity matrix with appropriately named indices so contraction results in a sum-reduction

Reply:
This has been added with this commit:
https://github.com/jcmgray/quimb/commit/7876b4fd7daf6a3b3a94e70d8370f55515b1d21d

But that was on 2020-08-20 and the current release is from 2020-02-18:
https://quimb.readthedocs.io/en/latest/changelog.html